app/root.tsx

```diff
@@ -7,13 +9,44 @@ import {
 	Scripts,
 	ScrollRestoration,
 	isRouteErrorResponse,
+	json,
 	useRouteError,
+	useRouteLoaderData,
 } from "@remix-run/react";
-
-import "@fontsource-variable/plus-jakarta-sans";
 import { ReactNode } from "react";
+import { fetchReports } from "~/server/impactReportHelpers";
 import "./tailwind.css";
 
+export const loader = async ({ request }: LoaderFunctionArgs) => {
```

Comment:
fetch `/impact-reports` instead - which does basically what this function does

Reply:
If you can directly call the impact report loader using `useLoaderData("impact-report")` then we don't need a loader in root

Reply:
the intention is to use the loader in `root` _instead_ of `impact-reports.tsx`, and delete that file. 
maybe i am still missing a piece but i am not able to get data into `_index` route from `impact-reports` route. using `fetch(/impact-reports)` doesn't trigger its loader. with the code from `impact-reports.tsx` moved into `root.tsx` i am able to use the data in `_index` route because the loader in`root` is already active.

Reply:
somehow missed this but I gave it a shot just now and I ran into the same issue! Turns out just fetching `/impact-reports` won't work as its an incomplete URL. Here's what worked:

```ts
export const loader = async ({ request }: LoaderFunctionArgs) => {
	const url = new URL(request.url);
	const response = await fetch(`${url.protocol}//${url.host}/impact-reports`);
	const data = await response.json();
	return json(data);
};
```

This allows the fetch to work dynamically on localhost and when its deployed.

Reply:
> const response = await fetch(`${url.protocol}//${url.host}/impact-reports`);

cool ✨
did you find this referenced in remix docs somewhere? i couldn't find it. i had tried hardcoding the full path couldn't get it to resolve.